csrc/cpu/utils.cpp

```diff
@@ -4,6 +4,9 @@
   #include <string>
   #include <sched.h>
 #endif
+#include <unistd.h>
+#include <sys/syscall.h>
+#define gettid() syscall(SYS_gettid)
```

Comment:
```suggestion
#if __GLIBC__ == 2 && __GLIBC_MINOR__ < 30
  #include <unistd.h>
  #include <sys/syscall.h>
  #define gettid() syscall(SYS_gettid)
#endif
```
Perhaps applying this usage to specific glibc versions is better.

Reply:
thanks you suggestion. i will update code.